### PR TITLE
Buildfix for Xcode7, iOS9

### PR DESCRIPTION
**Pull request by mgaver**
Buildfix for Xcode7, iOS9

Add icon library to linker flags
Relocate splash screen.(in root directory)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -788,6 +788,8 @@ elseif(IOS)
 	if(EXISTS "${CMAKE_IOS_SDK_ROOT}/System/Library/Frameworks/GameController.framework")
 		set(nativeExtraLibs ${nativeExtraLibs} "-weak_framework GameController")
 	endif()
+	
+	set(nativeExtraLibs ${nativeExtraLibs} iconv)
```
Comment (unknownbrackets) [inline CMakeLists.txt]: Should we just put this as an else for `if (ICONV_LIBRARY)`?  Apparently it's there.

-[Unknown]

**Comment by unknownbrackets** [comment]
What's the reasoning for the asset relocation?  Does it not work on iOS 9 without moving?

-[Unknown]

**Comment by mgaver** [comment]
Without moving the splash assets, there will be a letter-box on iPhone 5 or higher.(It works with 4:3 screen)

**Comment by unknownbrackets** [comment]
Okay, cool.  Thanks for moving it, I'm just going to make sure it builds on travis and then merge.

-[Unknown]

File: CMakeLists.txt (hunk 1 above)
Comment (mgaver) [inline CMakeLists.txt]: I added it.

**Pull request merged**